core/src/test/scala/org/apache/spark/sql/delta/expressions/InterleaveBitsSuite.scala

```diff
@@ -98,4 +103,26 @@ class InterleaveBitsSuite extends SparkFunSuite with ExpressionEvalHelper {
     InterleaveBits(Seq(Literal(0.toDouble))).checkInputDataTypes() != TypeCheckSuccess
     InterleaveBits(Seq(Literal(0.toString))).checkInputDataTypes() != TypeCheckSuccess
   }
+
+  test("randomization interleave bits") {
+    val numIters = sys.env
+      .get("NUMBER_OF_ITERATIONS_TO_INTERLEAVE_BITS")
```

Comment:
where is this env variable set?

Reply:
Currently, we can only set the env variable before we run the unit test.
Something like:
```
NUMBER_OF_ITERATIONS_TO_INTERLEAVE_BITS=20 ./build/sbt "testOnly *InterleaveBitsSuite"
```

Do you have a better idea to make this configurable?

Reply:
I think this ok. I wanted to check if there is another change to CI that sets this.